pkg/validator/types.go

```diff
@@ -2,6 +2,10 @@ package validator
 
 import (
 	"github.com/openservicemesh/osm/pkg/logger"
+	"github.com/pkg/errors"
 )
 
 var log = logger.New(ValidatorWebhookSvc)
+
+var errIngressBackendDuplicateBackends = errors.New("duplicate backends detected")
+var errUpstreamTrafficSettingDuplicateHosts = errors.New("duplicate hosts detected")
```

Comment:
Let's directly return the errors instead of statically defining this so we can add more context to the error message.

Reply:
These error vars are wrapped [later on](https://github.com/openservicemesh/osm/pull/4640/files#diff-ca09fc1c4167db01ac42e4c120198cbd797d234b280b9986266ac0440b016289R203) so as to add more context. The main reason for explicitly creating types here is so that we can use `errors.Is()` in order to [decide](https://github.com/openservicemesh/osm/pull/4640/files#diff-ca09fc1c4167db01ac42e4c120198cbd797d234b280b9986266ac0440b016289R164) whether or not  to fail the validating webhook

Reply:
Removed since validators were made to be deterministic